server.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+require('dotenv').config();
+const express = require('express');
+const app = express();
+const bodyParser = require('body-parser');
+const db = require('./db');
```

Comment:
The db should probably expose a initialize method.  This require doesn't make it clear that we just connected/setup the db.
Better example:
```
const db = require('./db');
...

db.setupAndConnect();
```

Reply:
db.setupAndConnect sounds very clear! done.